app/views/pieces/show.html.erb

```diff
@@ -1,32 +1 @@
-<h1> WELCOME TO YOUR GAME MWAHAHAHAH </h1>

```

Comment:
Done. I was on the fence about removing it at first since I initially thought it would cause conflict with the other pull request, but I think we can make changes accordingly.

Then.. I thought what if it improves smoothness? So I did it. :) Barely an improvement tho.